crates/primitives/src/user_operation.rs

```diff
@@ -405,6 +410,274 @@ pub struct UserOperationGasEstimation {
     pub call_gas_limit: U256,
 }
 
+fn ssz_pack_u256(
+    fixed: &mut Vec<Option<Vec<u8>>>,
+    fixed_lengths_sum: &mut usize,
+    variable_lengths: &mut Vec<usize>,
+    value: U256,
+) -> Result<(), ssz_rs::SerializeError> {
+    let mut element_buffer = Vec::with_capacity(32);
+    <[u64; 4] as ssz_rs::Serialize>::serialize(&value.0, &mut element_buffer)?;
+    fixed_lengths_sum.add_assign(32);
+    fixed.push(Some(element_buffer));
+    variable_lengths.push(0);
+    Ok(())
+}
+
+fn ssz_pack_bytes(
+    fixed: &mut Vec<Option<Vec<u8>>>,
+    fixed_lengths_sum: &mut usize,
+    variable: &mut Vec<Vec<u8>>,
+    variable_lengths: &mut Vec<usize>,
+    value: Bytes,
+) {
+    let size = value.len();
+    let mut element: Vec<u8> = Vec::with_capacity(size);
+    element.extend(value.iter());
+    fixed.push(None);
+    fixed_lengths_sum.add_assign(4);
+    variable_lengths.push(size);
+    variable.push(element);
+}
+
+impl ssz_rs::Sized for UserOperation {
+    fn is_variable_size() -> bool {
+        true
+    }
+    fn size_hint() -> usize {
+        0
+    }
+}
+
+impl ssz_rs::Serialize for UserOperation {
```

Comment:
@Vid201 I am handwriting this because I don't want to introduce another type of conversion. Do you have better ideas? Maybe we should wrap `Bytes`, `U256` these kinds of types in our crates?

Reply:
Hmm, I can't think of better solution at the moment. I think this is fine for now.

Should we add some 1 or 2 tests for ssz serialize/deserialize?

Reply:
Yes. We should